src/components/AppButton.js

```diff
@@ -0,0 +1,11 @@
+export const AppButton = (props) => {
```

Comment:
What is `AppButton` for? I am not sure I understand what the `App` part of the name refers to.

Reply:
True is, I don't need it, so I deleted it.